java/connector-node/risingwave-connector-service/src/main/resources/sql_server.properties

```diff
@@ -21,4 +21,5 @@ name=${hostname}:${port}:${database.name}.${schema.name}.${table.name:-RW_CDC_Sh
 # In sharing cdc mode, transaction metadata will be enabled in frontend.
 # For sql server, it's always false actually.
 provide.transaction.metadata=${transactional:-false}
-database.encrypt=false
+database.encrypt=${database.encrypt:-false}
+database.trustServerCertificate=true
```

Comment:
It is correct to always to `true` for this config? maybe we can default to `true` and allow passing different value.

Reply:
Ideally, we should allow users to either enable `trustServerCertificate` or provide a path to the cert. We can support it later.